PicPi/PhotosUpload.swift

```diff
@@ -7,30 +7,52 @@
 
 import Foundation
 import Moya
-
 /// photosUpload endpoint
 enum PhotosUpload {
-  ///enter nameAndExtension as one entry because when using PHAsset will return full file name with extension 
+  ///enter nameAndExtension as one entry because when using PHAsset will return full file name with extension
   case uploadPhoto(photoData: Data , nameAndExtension: String)
+  /// retrieve the URLs of all the thumbnails on the PI
+  case retrieveThumbnails
+  /// remove the specified photo by filename
+  case removePhoto(filename: String)
+  /// get the current preferences of the PI
+  case retrievePIPreferences
+  /// update the PI preferences
+  case updatePI(preferences: [String : Any])
 }
-
 extension PhotosUpload: TargetType {
   var baseURL: URL {
     ///Pi IP hard coded for now
-     return URL(string: "http://192.168.0.5:3000")!
+    return URL(string: "http://192.168.0.5:3000")!
   }
   /// path for the specified operation (e.g. : upload photo , delete photo...)
   var path: String {
     switch self {
       case .uploadPhoto:
-        return"/photos/upload"
+        return "/photos/upload"
+      case .retrieveThumbnails:
+        return "/photos"
+      case .removePhoto(let filename):
+        return "/photos/\(filename)"
+      case .retrievePIPreferences:
```

Comment:
wdyt of updating this to handle both cases like so:
```suggestion
      case .retrievePIPreferences, .updatePI:
```

Reply:
Lol didn't think of that